Sources/Proxy/Representations/SliceRepresentationProxy/index.js

```diff
@@ -255,10 +255,11 @@ function vtkSliceRepresentationProxy(publicAPI, model) {
     const bds = ds.getBounds();
     const axisIndex = 'XYZ'.indexOf(slicingMode);
     const endValue = bds[axisIndex * 2 + 1];
+    const spacing = ds.getSpacing()[axisIndex];
```

Comment:
That's not right: if you did IJK slicing, you could reuse the spacing values from the image data. When you are in XYZ, you can not just take that value if it's not aligned (worse: it could be along another direction).

I'll provide more insight and suggestions soon.

Reply:
Ah, good point. I was only testing with IJK slicing.

Reply:
`getClosestAxis` should help us here.